---
layout: post
title: Antibacterial Targets - Evidence for exclusion of targets for which host orthologues
  exist
date: '2013-09-22T12:22:00.002+01:00'
author: John Overington
tags:
- Some Of The Interesting Things You'll See On A Long Distance Flight
modified_time: '2013-09-22T19:59:37.214+01:00'
thumbnail: http://3.bp.blogspot.com/-liOtvnQP9kM/Uj7NzsH2J3I/AAAAAAAADIw/Iy4EZK-Zgd8/s72-c/Streptococcal_pharyngitis.jpg
blogger_id: tag:blogger.com,1999:blog-2546008714740235720.post-163045490792255602
blogger_orig_url: http://chembl.blogspot.com/2013/09/antibacterial-targets-evidence-for.html
---

<div class="separator" style="clear: both; text-align: center;">
<a href="http://3.bp.blogspot.com/-liOtvnQP9kM/Uj7NzsH2J3I/AAAAAAAADIw/Iy4EZK-Zgd8/s1600/Streptococcal_pharyngitis.jpg" imageanchor="1" style="margin-left: 1em; margin-right: 1em;"><img border="0" height="276" src="http://3.bp.blogspot.com/-liOtvnQP9kM/Uj7NzsH2J3I/AAAAAAAADIw/Iy4EZK-Zgd8/s320/Streptococcal_pharyngitis.jpg" width="320" /></a></div>
<br />
One of the classic mantras for the genomics-based discovery of novel anti-bacterials is to ignore targets for which orthologues exist in the host (human) genome, but my hunch was that the majority of antibacterial mechanisms have clear host orthologues (they do, as you'll see below). I haven't come across any really simple papers supporting this dogma in the past, so decided to have a quick look this morning. I am the unwilling host for an oral bacterial infection myself at the moment, but I must stress that the throat above is not mine, but an anonymous one from Teh Interweb!<br />
<br />
So, using a book I've just picked up at the ACS in Indy, I went through and did some quick analysis - the prose in the book is great, informal, and very very readable - buy it!<br />
<br />
<pre>%T Antibacterial Agents: Chemistry, Mode of Action, Mechanisms of Resistance and Clinical Applications
%A R.J. Anderson
%A P.W. Groundwater
%A A. Todd
%A A.J. Worsley
%I Wiley
%D 2012
%O ISBN 978-0-470-97245-8
</pre>
<br />
I went through, and at a drug class level, assigned the distinct mechanisms into three target classes.<br />
<ol>
<li>Orthologue of antibacterial target is present in humans.</li>
<li>Orthologue of antibacterial target is absent in humans.</li>
<li>Antibacterial acts through a non gene-derived target mechanism.</li>
</ol>
The counts for these three states are 8, 4, 3 or in a simple graphical form<br />
<br />
<div class="separator" style="clear: both; text-align: center;">
<a href="http://2.bp.blogspot.com/-REwB1kwGDms/Uj7P3ispqTI/AAAAAAAADI8/WeJnZHKbvIo/s1600/antibac_targets.tiff" imageanchor="1" style="margin-left: 1em; margin-right: 1em;"><img border="0" height="256" src="http://2.bp.blogspot.com/-REwB1kwGDms/Uj7P3ispqTI/AAAAAAAADI8/WeJnZHKbvIo/s400/antibac_targets.tiff" width="400" /></a></div>
<br />
So, no real great evidence to focus on bacteria specific genes - in fact it's 2:1 in favour of targets for which orthologues exist in the host. The key would seem to be more exploiting physicochemical differences between bacterial and human cells (<i>e.g.</i> acidity), or exploit differences in the binding sites. I guess the dogma arose for a couple of reasons - firstly everyone knowns about penicillins, and secondly, it is an easy filter to apply bioinformatically, and finally it just seems like a perfectly sensible thing to do with respect to elimination of mechanism-based toxicity - and so is often done.<br />
<br />
On this latter point, that of mechanism-based toxicity, this can be really important, but remember, a drug dosed to a human is not magically attracted only to relatives of the bacterial target, it will sample and equilibrate across all accessible binding sites of all proteins, and drugs will have side-effects and toxicity related to 1) binding to orthologues, 2) binding to paralogues, and 3) binding to anything else. A nice example of this is the paper from <i>Science</i> earlier this year on the side effects of sulphonamide antibacterials via inhibition of host<span style="font-family: inherit;">&nbsp;<span style="background-color: white;">sepiapterin reductase</span></span><span style="background-color: white; font-family: Verdana, Arial, Helvetica, sans-serif; font-size: 12px;">.</span><br />
<br />
To be clear what I did here. Firstly - I used the chapters in the Antibacterial Agents book to define a class, so the graph can be plotted in many other ways - however, I was interested in distinct mechanisms. Secondly, several antibacterials don't target proteins, but various parts of the ribosome - these are gene derived, so count above as gene products, but they are not proteins (well they are riboproteins). Even if you strip these RNA targets out though (there are 5) the numbers are still not compelling for the need to avoid host target orthologues - the ratio would be 3:4 instead of 8:4.<br />
<br />
What are the implications of removing this bacterial specific filter? Well that is more than a quick job on a Sunday morning and two cups of Lapsong Suchong - but my feeling is it might be quite significant.<br />
<br />
jpo