---
layout: post
title: 'Paper: PPDMs â€“ A resource for mapping small molecule bioactivities from ChEMBL
  to Pfam-A protein domains'
date: '2014-10-28T07:16:00.001Z'
author: John Overington
tags:
- ChEMBL
- ChEMBL Curation Notes
- Books and Papers
modified_time: '2014-10-28T07:16:08.135Z'
thumbnail: http://3.bp.blogspot.com/-2CmKMhHQJz8/VE9BBEK8FGI/AAAAAAAADf8/iYxE1lyKGnY/s72-c/ppdms.tiff
blogger_id: tag:blogger.com,1999:blog-2546008714740235720.post-1109309691877778873
blogger_orig_url: http://chembl.blogspot.com/2014/10/paper-ppdms-resource-for-mapping-small.html
---

<div class="separator" style="clear: both; text-align: center;">
<a href="http://3.bp.blogspot.com/-2CmKMhHQJz8/VE9BBEK8FGI/AAAAAAAADf8/iYxE1lyKGnY/s1600/ppdms.tiff" imageanchor="1" style="margin-left: 1em; margin-right: 1em;"><img border="0" src="http://3.bp.blogspot.com/-2CmKMhHQJz8/VE9BBEK8FGI/AAAAAAAADf8/iYxE1lyKGnY/s1600/ppdms.tiff" height="400" width="308" /></a></div>
<br />
We've just published a Open Access <a href="http://bioinformatics.oxfordjournals.org/content/early/2014/10/26/bioinformatics.btu711.full.pdf+html">paper in Bioinformatics</a> on an approach to annotate the region of ligand binding within a target protein. This has a lot of applications in the use of <a href="http://www.ebi.ac.uk/chembl">ChEMBL</a>, in particular providing greater accuracy in mapping functional effects, improving ligand-based target prediction approaches, and reducing false positives in sequence/target searching of ChEMBL. Where next for this work - well annotating to a site-specific level would be a good thing to implement (think about HIV-1 RT with the distinct nucleoside and non-nucleoside sites).
<br />
<div>
<br /></div>
<div>
Here's the abstract...</div>
<div>
<br /></div>
<b>Summary</b>: PPDMs is a resource that maps small molecule bioactivities to protein domains from the Pfam-A collection of protein families. Small molecule bioactivities mapped to protein domains add important precision to approaches that use protein sequence searches alignments to assist applications in computational drug discovery and systems and chemical biology. We have previously proposed a mapping heuristic for a subset of bioactivities stored in ChEMBL with the Pfam-A domain most likely to mediate small molecule binding. We have since refined this mapping using a manual procedure. Here, we present a resource that provides up-to-date mappings and the possibility to review assigned mappings as well as to participate in their assignment and curation. We also describe how mappings provided through the PPDMs resource are made accessible through the main schema of the ChEMBL database.<br />
<br />
<b>Availability</b>:&nbsp;The PPDMs resource and curation interface is available at&nbsp;<a href="https://www.ebi.ac.uk/chembl/research/ppdms/pfam_maps">https://www.ebi.ac.uk/chembl/research/ppdms/pfam_maps</a><br />
<div id="p-4" style="border: 0px; line-height: 1.5; margin-bottom: 10px; outline-style: none; padding: 0px; vertical-align: baseline; width: 510.828125px;">
<span style="font-family: inherit;"><br /></span>
<span style="font-family: inherit;">The source-code for PPDMs is available under the Apache license at&nbsp;<a href="https://github.com/chembl/pfam_maps">https://github.com/chembl/pfam_maps</a>.&nbsp;</span></div>
<br />
Source code is available at&nbsp;<a href="https://github.com/chembl/pfam_map_loader">https://github.com/chembl/pfam_map_loader</a>&nbsp;to demonstrate the integration process with the main schema of ChEMBL.
