---
layout: post
title: Target Validation Using Genetics Approaches
date: '2013-02-11T10:00:00.003Z'
author: John Overington
tags: 
modified_time: '2013-02-11T10:00:59.739Z'
blogger_id: tag:blogger.com,1999:blog-2546008714740235720.post-9031099804527900341
blogger_orig_url: https://www.blogger.com/comment.g?blogID=2546008714740235720&postID=9031099804527900341
---

Target validation - the definitive experiment that links modulation of a target with effect on a disease, is now considered to be one of the rate and productivity limiting factors in the discovery of new medicines. Arguably, this is made more important due to the desire for new modes of action, and the fact that we just know so much more about ADMET than we used to, and so more experimental medicines are getting decent exposure <i>in homo</i>. An additional major challenge is that animal models either do not exist, or are not sufficiently predictive of human pathology.<br />
<br />
Of course, the absolutely definitive experiment is performed with a drug, but this is slow and expensive, and it is difficult to cast this in a way that doesn't look a lot like 'full on' drug discovery - Public-Private Partnerships - such as the great work of the SGC and their associates are showing the way on the discovery of Open pharmacological tools.<br />
<br />
As one attack on this problem, we have recently become involved in a collaboration using natural human population variation as one way of looking at target validation, and the results so far look very promising. We have a draft paper on the approach at late draft stage, so hopefully this will be out soon, but I thought it may be useful to outline the approach here....<br />
<br />
Probably everyone is familiar with monogenic genetic diseases, not all genetic diseases are heritable - but the majority are. Wikipedia has a <a href="http://en.wikipedia.org/wiki/Genetic_disorder">great introduction</a> to some of the complexities of dominant, recessive, linkage and so forth. The simplest case to imagine is where both copies of an inherited gene are defective, and the gene function (usually a misfolded or inactive expressed protein) is absent. An example many will be familiar with is the autosomal recessive disease phenylketonuria (PKU) - where the PAH gene is inactive in a number of different natural variants. (Further discussion of this system <a href="http://en.wikipedia.org/wiki/Dominance_(genetics)">here</a>). For the parents amongst you, testing for PKU is performed soon after birth in many countries with a blood sample taken from the heel of the newborn. If PKU is not recognised, high plasma levels of phenylalanine cause mental retardation.<br />
<br />
The PKU gene comes in a number of variants (alleles), usually labelled A, B, C, etc., and since we have two copies of this gene, they are partitioned in humans in the following ways, AA, AB, AC, BB, BC, CC, etc., in frequencies which reflect the underlying distribution of the parent alleles in the population. A is fully active, and the most frequent, so children that inherit AA have fully functioning phenylalanine metabolism. The B allele is inactive, and so, due to dose effects, someone with the AB genotype have one inactive and one working copy. Their plasma phenylalanine levels are higher than those with the AA genotype, but not sufficient to cause clinical problems - the body is quite robust to this sort of thing (isn't evolution wonderful!). For those children who inherit two copies of the inactive B PAH gene, and so have the BB phenotype, have no competant system to metabolise phenylalanine, and so suffer from PKU.<br />
<br />
This is a simple system, well understood and treated. A key point in the above is that the plasma levels of phenylalanine track the genetic allele assignment. AA - low phe level, AB - intermediate phe level, BB - high phe level. So, if we profiled the serum of a lot of patients, and we knew that high phenylalanine levels were toxic, we could get a lot of information by genotyping people, and combining this with observed phenylalanine levels.<br />
<br />
There are a whole range of genetic variants within a gene, ranging from mutations that lead to a dead gene at a transcript level (so the gene is not copied into RNA) to mutations where there are effects on the amino acids in the translated gene product, and more usually with no effect on the protein structure. However, there are many mechanisms through which mutations can affect gene function - for example, the protein may still be active, but be unstable and so is cleared from the body quicker, and so there is less activity than normal - the same effects can happen at a mRNA level, with less stability and faster clearance, and consequently less protein produced. It is also the case that mutations can actually make a protein more active, but as in most of life, it is usually easier to break something than to make it better.<br />
<br />
Given that these alleles are assigned randomly at birth, and are constant throughout ones life.<br />
<br />
<br />
<br />
<br />